lib/hanami/commands/console.rb

```diff
@@ -22,7 +22,7 @@ def reload!
         'irb'  => 'IRB'
       }.freeze
 
-      DEFAULT_ENGINE = 'irb'.freeze
+      DEFAULT_ENGINE = ['irb'].freeze
```

Comment:
@railsme I'd go for your array solution because of the perf gain over the others. I'll wait for another round of review from @jodosha and @AlfonsoUceda